docs/content/manual/pre-release/resiliency/simulated-slow-disk.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "[#2206](https://github.com/longhorn/longhorn/issues/2206) Fix the spinning disk on Longhorn"
+---
+
+This case requires the creation of a slow virtual disk with `dmsetup`.
+1. Make a slow disk:
+  - Make a disk image file: `truncate -s 10g slow.img`
+  - Create a loopback device: `losetup --show -P -f slow.img`
+  - Get the block size of the loopback device: `blockdev --getsize /dev/loopX`
+  - Create slow device: `echo "0 <blocksize> delay /dev/loopX 0 500" | dmsetup create dm-slow`
+  - Format slow device: `mkfs.ext4 /dev/mapper/dm-slow`
+  - Mount slow device: `mount /dev/mapper/dm-slow /mnt`
+2. Build longhorn-engine and run it on the slow disk.
+  - `make` or use a tag from docker hub in next step.
+  - `docker run --privileged -v /dev:/host/dev -v /proc:/host/proc -v /mnt:/volume longhornio/longhorn-engine:<tag> launch-simple-longhorn slow-vol 10g tgt-blockdev`
+3. Peform intense I/O and verify that it doesn't fail.
+  - This uses an iodepth of 16 to maximize the number of threads in tgtd: `fio --name=random-writers --ioengine=libaio --rw=randwrite --bs=1280k --direct=1 --size=3000m --numjobs=1 --filename=/dev/longhorn/slow-vol --iodepth=16`
```

Comment:
In fact any IODepth more than 16 should maximize the number of threads in tgtd.

Reply:
I noted that 16 or more with maximum the threads.